### PR TITLE
fix(dashboard): allow to unset currency cell value

### PR DESCRIPTION
**Pull request by fPolic**
**What**
- unset datagrid currency cell on delete press instead of setting it to 0
- consolidate pricing editors validations
- fix PL edit pricing schema

---

FIXES CC-529

### CODE REVIEW

File: packages/admin/dashboard/src/routes/locations/location-service-zone-shipping-option-create/components/create-shipping-options-form/create-shipping-options-form.tsx
Comment (srindom) [inline packages/admin/dashboard/src/routes/locations/location-service-zone-shipping-option-create/components/create-shipping-options-form/create-shipping-options-form.tsx:1]: q(non-blocking): should a similar thing happen on editing shipping option prices?

The behavior seems to be correct when testing it but wanted to put it here to be sure.

**Review by srindom — APPROVED** [review]
LGTM!

File: packages/admin/dashboard/src/routes/locations/location-service-zone-shipping-option-create/components/create-shipping-options-form/create-shipping-options-form.tsx
Comment (fPolic) [reply packages/admin/dashboard/src/routes/locations/location-service-zone-shipping-option-create/components/create-shipping-options-form/create-shipping-options-form.tsx:1]: Good point, will add the same check there for consistency.

Comment (fPolic) [reply packages/admin/dashboard/src/routes/locations/location-service-zone-shipping-option-create/components/create-shipping-options-form/create-shipping-options-form.tsx:1]: Nvm, we had the check there before, should be consistent now across all pricing editors!

**Pull request merged**